apis/fluent/atrium-api-fluent/src/commonMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/listFeatureExtractors.kt

```diff
@@ -26,3 +27,28 @@ fun <E, T : List<E>> Expect<T>.get(index: Int): Expect<E> =
 fun <E, T : List<E>> Expect<T>.get(index: Int, assertionCreator: Expect<E>.() -> Unit): Expect<T> =
     _logic.get(index).collectAndAppend(assertionCreator)
 
+/**
+ * Expects that list is not empty and returns an [Expect] for the last element in list.
+ *
+ * @return The newly created [Expect] for last element.
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.ListFeatureExtractorSamples.lastFeature
+ *
+ * @since 1.2.0
+ */
+val <E, T : List<E>> Expect<T>.last: Expect<E>
+    get() = _logic.last().transform()
+
+/**
+ * Expects that the subject of `this` expectation (a [List]) is not empty and that the last element
+ * in the list holds all assertions the given [assertionCreator] creates for it.
+ *
+ *
+ * @return an [Expect] for the subject of `this` expectation.
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.ListFeatureExtractorSamples.last
+ *
+ * @since 1.2.0
+ */
+fun <E, T : List<E>> Expect<T>.last(assertionCreator: Expect<E>.() -> Unit): Expect<T> =
```

Comment:
:+1: that what I meant, move last with `assertionCreator` also to this file

Reply:
Nice, thank you. So I guess it's ready to be checked by workflows.